### PR TITLE
Update LocationTime component

### PR DESCRIPTION
**Pull request by soumalyapakrashi**
LocationTime component now fetches current date and time in local timezone if no props are provided. To display time in another timezone, the new timezone needs to be provided and component will calculate time and date accordingly.

### CODE REVIEW

**Comment by soumalyapakrashi** [comment]
The component updates time according to the system time. It also shows time in different timezones accordingly.

File: src/components/LocationTime/LocationTime.js [hunk 1]
```diff
@@ -1,12 +1,33 @@
...
+    timezone = "local",
+    alignment = "left",
 }) {
+    let time = null;
+    let day = null;
+    let date = null;
+    
+    // By default, as local timezone is selected, we create date in local time
+    // let dt = DateTime.local();
+    let [ dt, setDateTime ] = useState(DateTime.local());
```
Comment (debaleenatalukdar98) [inline src/components/LocationTime/LocationTime.js]: make it const 
rename it as dt and setDt or dateTime and setDateTime

File: src/components/LocationTime/LocationTime.js [hunk 2]
```diff
@@ -1,12 +1,33 @@
...
+
+    // For timezones other than local
+    if(timezone !== "local") {
+        dt = dt.setZone(timezone);
```
Comment (debaleenatalukdar98) [inline src/components/LocationTime/LocationTime.js]: setDateTime(prevState => prevState.setZone(timeZone))

File: src/components/LocationTime/LocationTime.js [hunk 3]
```diff
@@ -1,12 +1,33 @@
...
+    setTimeout(() => {
+        setDateTime(DateTime.local());
+    }, 1000);
+
+    time = dt.toLocaleString(DateTime.TIME_24_SIMPLE);
+    day = dt.toLocaleString({ weekday: 'long' });
+    date = dt.toLocaleString({ day: 'numeric', month: 'short' })
+
     return(
         <div className={ alignment && alignment === "right" ? "location-time flex-align-end" : "location-time flex-align-start" }>
```
Comment (debaleenatalukdar98) [inline src/components/LocationTime/LocationTime.js]: alignment && not required

`{location-time ${alignment === "right" ? "flex-align-end" : "flex-align-start"}}`

File: src/components/Navbar/Navbar.js [hunk 4]
```diff
@@ -5,10 +5,7 @@ export function Navbar() {
     return(
         <nav>
             <LocationTime
-                time={ "18:00" }
-                location={ "Minsk" }
-                day={ "Friday" }
-                date={ "26 Nov" }
+                location={ "Pune" }
```
Comment (debaleenatalukdar98) [inline src/components/Navbar/Navbar.js:8]: add conditionally

File: src/components/TimeZoneCard/TimeZoneCard.js [hunk 5]
```diff
@@ -30,10 +27,8 @@ export function TimeZoneCard({
                         <h4>{ country }</h4>
                     </div>
                     <LocationTime
-                        time={ time }
-                        day={ day }
-                        date={ date }
                         alignment={ "right" }
+                        timezone={ "Australia/Sydney" }
```
Comment (debaleenatalukdar98) [inline src/components/TimeZoneCard/TimeZoneCard.js:31]: add conditoinally

File: src/components/LocationTime/LocationTime.js (hunk 1 above)
Comment (soumalyapakrashi) [reply src/components/LocationTime/LocationTime.js]: Done

File: src/components/LocationTime/LocationTime.js (hunk 2 above)
Comment (soumalyapakrashi) [reply src/components/LocationTime/LocationTime.js]: Updated

File: src/components/LocationTime/LocationTime.js (hunk 3 above)
Comment (soumalyapakrashi) [reply src/components/LocationTime/LocationTime.js]: Updated

File: src/components/Navbar/Navbar.js (hunk 4 above)
Comment (soumalyapakrashi) [reply src/components/Navbar/Navbar.js:8]: Will update after adding current location fetch function.

File: src/components/TimeZoneCard/TimeZoneCard.js (hunk 5 above)
Comment (soumalyapakrashi) [reply src/components/TimeZoneCard/TimeZoneCard.js:31]: Will update after adding the timezone api to fetch timezone.

**Review by debaleenatalukdar98 — APPROVED** [review]

**Pull request merged**